# Cole Southard

# Question 1

In [ ]:
def dist(x1,y1,x2,y2):
    return ((x2-x1)**2 + (y2-y1)**2)**(1/2)

answer1 = str(dist(3,4,7,10))
print(f'The distance between the two points is {answer1}')

The distance between the two points is 7.211102550927978


# Question 2

In [ ]:
def manhatDist(x1,y1,x2,y2):
    return (abs(x1-x2) + abs(y1-y2))

answer2 = str(manhatDist(1,5,5,8))
print(f'The Manhattan distance between the two points is {answer2}')

The Manhattan distance between the two points is 7


# Question 3

In [ ]:
def ptLineDist(x,y,a,b,c):
    return (abs(a*x + b*y + c)/(a**2 + b**2)**(1/2))

answer3 = str(ptLineDist(4,7,2,-4,-1))
print(f'The distance between the point and the line is {answer3}')

The distance between the point and the line is 4.695742752749558


# Question 4

In [ ]:
import math

def combination(n,r):
    return ((math.factorial(n)) / (math.factorial(r) * (math.factorial(n-r))))

answer4 = str(combination(52,2))
print(f'You can be dealt {answer4} private hands in a game of Texas Hold\'em')

You can be dealt 1326.0 private hands in a game of Texas Hold'em


# Question 5

In [ ]:
def windLoading(rho,u,cd,a):
    return (.5 * rho * u**2 * cd * a)

answer5 = str(windLoading(.00237,15*1.467,2,20*30))
print(f'The force on the wall is {answer5} pounds.')

The force on the wall is 688.5608755500002 pounds.


# Question 6

In [1]:
from math import e,log
answer6 = round((e ** 7) / (log(9,1.5) + log(10,2) + log(11) + log(12,3)),3)

print(answer6)

81.834


# Question 7

In [6]:
from math import e, pi

def sinh(x):
    return (e**x - e**(-x))/2

answer7 = sinh(pi/2)
print(answer7)


2.3012989023072947


# Question 8

In [52]:
def futureValue(p,r,t):
    return (p * (1+r)**t)

rate = .08

investor1tot = 0
for i in range(25,65,10):
    investor1tot += futureValue(10000,rate,65-i)

investor2tot = 0
for i in range(35,65,10):
    investor2tot += futureValue(20000,rate,65-i)

investor1gain =  investor1tot-40000
investor2gain =  investor2tot-60000

answer8 = investor1gain/investor2gain

print('At retirement, investor 1 has $' + str(round(investor1tot,2)))
print('At retirement, investor 2 has $' + str(round(investor2tot,2)))
print('')
print('Investor 1 gained $' + str(round(investor1gain,2)))
print('Investor 2 gained $' + str(round(investor2gain,2)))
print('')
print('Investor 1 made ' + str(round(answer8,3)) + ' times more capital gain than investor 2')



At retirement, investor 1 has $386070.61
At retirement, investor 2 has $337650.78

Investor 1 gained $346070.61
Investor 2 gained $277650.78

Investor 1 made 1.246 times more capital gain than investor 2


# Question 9

In [60]:
def futureValue(p,r,t):
    return (p * (1+r)**t)

rate = .08

investor3tot = 0
for i in range(18,65):
    investor3tot += futureValue(1000,rate,65-i)
# investor 3 starts at age 18, contributing $1000 per year

investor4tot = 0
for i in range(18,65):
    investor4tot += futureValue(1000,rate+.01,65-i)

investor3gain =  investor3tot - (1000*(65-18))
investor4gain =  investor4tot - (1000*(65-18))

answer9 = investor4gain/investor3gain

print('At retirement, investor 3 has $' + str(round(investor3tot,2)))
print('At retirement, investor 4 has $' + str(round(investor4tot,2)))
print('')
print('Investor 3 gained $' + str(round(investor3gain,2)))
print('Investor 4 gained $' + str(round(investor4gain,2)))
print('')
print('Investor 4 made ' + str(round(answer9,3)) + ' times more capital gain than investor 3')


At retirement, investor 3 has $489132.16
At retirement, investor 4 has $683280.41

Investor 3 gained $442132.16
Investor 4 gained $636280.41

Investor 4 made 1.439 times more capital gain than investor 3


The first investor made $68,419.83 more than investor 1 in question 8.

It's wise to make contributions as soon as possible so the gains have more time to compound. The more time in the market the better.

The interest rate has a massive impact on the total gains made. Investor 4 had an annual rate of return just 1 percent greater than investor 3, but made 1.44 times more money over the life of the investment with all other factors being equal.